openwisp-utils-qa-checks

```diff
@@ -151,3 +162,5 @@ fi
 if ! $SKIP_CHECKCOMMIT; then
   runcheckcommit
 fi
+
+exit $FAILURE
```

Comment:
Can you please change it to 
```
    if [[ $TRAVIS = 1 ]]; then
        exit $FAILURE
    fi
```

Why? -- Because if I am running these tests on my personal machine, most likely I don't want to exit after the tests. (It'll be quite irritating to open another terminal and activate `venv` again and again, wouldn't you agree? :joy: )

Reply:
Actually, I'll resolve this one because the script can run without exiting the terminal. 
( Sorry, I was thinking about the `docker-openwisp` script which required this when I was making it because I needed to import those test functions to run them during development! )